source/clear-linux/reference/bundles/openssh-server.rst

```diff
@@ -8,6 +8,46 @@ Remote users require a SSH service to be able to use an encrypted login
 shell. The first time OpenSSH starts, it generates the server SSH keys needed
 for the service.
 
+|CL| enables the `sshd.socket` unit, which will listen on port 22 by default and
+start the openssh service as required.
+
+Change Default Port
```

Comment:
Update section heading to: "Change default port" (use sentence case)
Update underline to use *